build/scripts/meta.yaml.schema

```diff
@@ -34,7 +34,7 @@
     },
     "icon": {
       "type": "string",
-      "pattern": "[-._a-zA-Z0-9]+.svg[^.]*$"
+      "pattern": "[-._a-zA-Z0-9]+(.svg|.png)[^.]*$"
```

Comment:
The periods in the filename need to be escaped, otherwise this pattern will match names like `filepng` or `filesvg`.

`(.svg|.png)` -> `(\.svg|\.png)`

Reply:
Updated with following
`[-._a-zA-Z0-9]+?\.(?:svg|png)(?:\?.*)?$`